src/util.py

```diff
@@ -11,10 +11,8 @@
 db_config = {'user': 'root',
              'password': 'root',
              'db_host': 'localhost',
-
-             'db_name': 'psdProject'}
-
              'db_name': 'psd_project'}
-cluster_model_file = {'Chinese': '/home/zglg/SLU/psd/cluster_pre_train/gensim-word2vec-model-Chinese',
-                      'English': '/home/zglg/SLU/psd/cluster_pre_train/gensim-word2vec-model-English'}
+
```

Comment:
We need to add a user file input function here, and everywhere else that calls a file.